lib/Widgets/Settings.vala

```diff
@@ -96,6 +115,39 @@ namespace Granite {
             }
         }
 
+        private void setup_accent_color () {
+            try {
+                portal = Portal.Settings.get ();
+
+                var variant = portal.read (
+                    "org.freedesktop.appearance",
+                    "accent-color"
+                ).get_variant ();
+
+                accent_color = parse_color (variant);
+
+                portal.setting_changed.connect ((scheme, key, value) => {
```

Comment:
@alice-mkh maybe it's because I'm on an older version of some library but I'm not getting anything at all here when I change the color in system settings so I can't really test further because it's broken for me all the way at the beginning

Reply:
Huh okay. It did work for me, but only until I open inspector due to the global provider issue I mentioned.

Reply:
Yeah I can't even test that :sweat_smile: